src/places/queries.js

```diff
@@ -51,3 +53,19 @@ export function usePlaceListQuery (queryOptions = {}) {
     places: query.data,
   }
 }
+
+export function usePlaceTypeListQuery (queryOptions = {}) {
+  const query = useQuery(
+    queryKeyPlaceTypeListAll(),
+    () => placeTypeAPI.list(),
+    {
+      placeholderData: () => [],
+      staleTime: Infinity,
```

Comment:
Could rjiggle this to cause them to be reloaded now and again, given there are no socket updates... (if I understood correctly). Not sure the best options for that.